posthog/test/base.py

```diff
@@ -351,14 +348,28 @@ def setUp(self):
         cache.clear()
         TEST_clear_instance_license_cache()
 
-        # Sets the cloud mode to stabilise things tests, especially num query counts
-        # Clear the is_rate_limit lru_Caches so that they does not flap in test snapshots
+        # Sets the cloud mode to stabilize things tests, especially num query counts
+        # Clear the is_rate_limit lru_Caches so that they do not flap in test snapshots
```

Comment:
flyby grammar policing 🚓

Reply:
Americans 💪 😅  (but yes thank you our default is American English no matter how much British English I try to sneak in 😂 )